tests/integration/installation_test.go

```diff
@@ -139,10 +143,21 @@ func debugFailedInstallation(ctx context.Context, inst *porterv1.Installation) {
 	Log("DEBUG: ----------------------------------------------------")
 }
 
+// Get the amount of time that we should wait for a test action to be processed.
+func getWaitTimeout() time.Duration {
+	if value := os.Getenv("PORTER_TEST_WAIT_TIMEOUT"); value != "" {
+		timeout, err := time.ParseDuration(value)
+		if err == nil {
+			return timeout
+		}
+	}
```

Comment:
Maybe we can add a log or something to indicate the provided value isn't being used here due to parsing error?

Reply:
Fixed!